frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -158,9 +158,9 @@ export const TicketsTable = ({ isNewTicketDirty, setTicketValue }: TicketsTableP
 
 	useEffect(() => {
 		visibleSortedColumnsNames
-			.filter((name) => !INITIAL_COLUMNS.includes(name))
+			.filter((name) => !INITIAL_COLUMNS_NO_OVERRIDES.includes(name))
 			.forEach((name) => fetchColumn(name, filteredTickets));
-	}, [filteredTickets.length, visibleSortedColumnsNames.join('')]);
+	}, [filteredTickets.map(({ _id }) => _id), visibleSortedColumnsNames.join('')]);
```

Comment:
can you use directly filteredTickets? the mapping will create a new object each time so this will be triggered more times than necessary.
the filteredTickets object on the other hand changes only when tickets or filteredids changes which in turn changes only when fetching the tickets or when changing the filters.